docs/root/intro/arch_overview/other_features/ip_transparency.rst

```diff
@@ -45,13 +45,14 @@ metadata includes the source IP. Envoy supports consuming this information using
 the downstream remote address for propagation into an
 :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>` header. It can also be used in
 conjunction with the
-:ref:`Original Src Listener Filter <arch_overview_ip_transparency_original_src_listener>`.
+:ref:`Original Src Listener Filter <arch_overview_ip_transparency_original_src_listener>`. Finally,
+Envoy supports generating this header using the :ref:`Proxy Protocol Transport Socket <extension_envoy.transport_sockets.upstream_proxy_protocol>`
+transport socket.
```

Comment:
Can we have a full example config here?

Reply:
Yeah, I was thinking about that, but was unsure where to add. We don't really have a designated socket sport for config examples. At one point I thought we could maybe add a sockets section here https://www.envoyproxy.io/docs/envoy/latest/configuration/configuration? wdyt? I'm fine with just adding in this file.

Reply:
@lizan